tests/load/locustfiles/locustfile.py

```diff
@@ -123,6 +139,31 @@ def hello(self) -> None:
             timer.response_length = len(reply.encode("utf-8"))
         self.uaid = res.uaid
 
+    def ack(self) -> None:
+        """
+        Send an 'ack' message to push.
+
+        After sending a notification, the client must also send an 'ack' to the server to
+        confirm receipt. If there is a pending notification, this will try and receive it
+        before sending an acknowledgement.
+        """
+        print(f"notification")
+        with self._time_event(name="acknowledge") as timer:
+            reply = self.ws.recv()
+            notification = NotificationMessage(**json.loads(reply))
+            body = json.dumps(
+                dict(
+                    messageType="ack",
+                    use_webpush=True,
+                    updates=dict(channelID=notification.channelID, version=notification.version)
```

Comment:
The register failing is due to the `ack` being malformed, causing autopush to not be in a happy state when the register was issued:
```suggestion
                    messageType="ack",
                    updates=[dict(channelID=notification.channelID, version=notification.version)]
```